Lottery.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public static Ball getRandomBall() {

```

Comment:
Method should not be a static
```suggestion
    public Ball getRandomBall() {

```